wd_helper.au3

```diff
@@ -1115,7 +1115,8 @@ Func _WD_ElementSelectAction($sSession, $sSelectElement, $sCommand, $aParameters
 	EndIf
 
 	Local $sMessage = '$sCommand = ' & $sCommand & ' : Result = ' & ((IsArray($vResult)) ? ("(array)") : ($vResult))
-	Return SetError(__WD_Error($sFuncName, $iErr, $sMessage), $_WD_HTTPRESULT, $vResult)
+	If Not $iExt Then $iExt = $_WD_HTTPRESULT
+	Return SetError(__WD_Error($sFuncName, $iErr, $sMessage, $iExt), $iExt, $vResult)
```

Comment:
This situation crossed my mind earlier today. How is the calling routine to know if the value in @extended is the HTTP result or a diagnostic code? 

We probably should eliminate setting @extended = $_WD_HTTPRESULT and have them call _WD_LastHTTPResult instead when needed.

Reply:
Going to bed. Will look again tomorrow.

Reply:
`_WD_LastHTTPResult` moved from `wd_helper.au3` to `wd_core.au3`

Reply:
> `_WD_LastHTTPResult` moved from `wd_helper.au3` to `wd_core.au3`

Should this be also mentioned in some kind of header or list of functions?

Reply:
OIC I see what you were saying here. I'm not sure how best to document this. Will need to think about it.

Reply:
OIC ??